tests/test_resources/test_modules/test_functions/test_aws_lambda/test_aws_lambda.py

```diff
@@ -0,0 +1,554 @@
+import logging
+import os
+import unittest
+from pathlib import Path
+
+import boto3
+import pytest
+
+import runhouse as rh
+
+logger = logging.getLogger(__name__)
+CUR_WORK_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_RESOURCES = f"{CUR_WORK_DIR}/test_helpers/lambda_tests"
+
+CRED_PATH = f"{Path.home()}/.aws/credentials"
+DEFAULT_REGION = "us-east-1"
+
+if Path(CRED_PATH).is_file():
+    LAMBDA_CLIENT = boto3.client("lambda")
+else:
+    LAMBDA_CLIENT = boto3.client("lambda", region_name=DEFAULT_REGION)
+IAM_CLIENT = boto3.client("iam")
+LAMBDAS_NAMES = set()
```

Comment:
Do we still need this? We should allow for running each test standalone, and not rely on maintaining a global set of active functions. We could automatically teardown the function at end of a test (or add a flag for that, similar to what we do with running docker containers in detached mode)